app/_src/gateway/production/deployment-topologies/hybrid-mode/index.md

```diff
@@ -15,7 +15,38 @@ of the CP nodes, and only the CP nodes are directly connected to a database.
 Instead of accessing the database contents directly, the DP nodes maintain a 
 connection with CP nodes to receive the latest configuration.
 
-![Hybrid mode topology](/assets/images/products/gateway/deployment-hybrid-2.png)
+<!--vale off -->
+{% mermaid %}
+flowchart LR
+
+A[(Database)]
```

Comment:
I think the use of 'kong gateway instance' isn't as helpful as Data plane 1, 2, 3 and control plane.